body-parser.ts

```diff
@@ -0,0 +1,21 @@
+import * as Koa from 'koa';
+import * as parse from 'co-body';
+
+declare module 'koa' {
```

Comment:
I forget why this was necessary... are we using an old version of koa without types published somewhere? What breaks if we remove this?

Reply:
Apparently the Koa `Context` object doesn't have a `body` type. Not sure why. This is using the latest `@types/koa` and the latest `koa`.

Here is the error:

```
(2339) Property 'body' does not exist on type 'Request'.
```

Reply:
Ohh, it's because we attach that ourselves in `body-parser.ts` I think. Should double-check the docs and make sure it's OK to put stuff in the ctx.request namespace